botx/models/message/incoming_message.py

```diff
@@ -251,17 +252,20 @@ def to_domain(self, raw_command: Dict[str, Any]) -> IncomingMessage:  # noqa: WP
         forward: Optional[Forward] = None
         reply: Optional[Reply] = None
         for entity in self.entities:
-            entity_domain = convert_bot_api_entity_to_domain(entity)
-            if isinstance(entity_domain, Mention):
-                mentions.append(entity_domain)
-            elif isinstance(entity_domain, Forward):
-                # Max one forward per message
-                forward = entity_domain
-            elif isinstance(entity_domain, Reply):
-                # Max one reply per message
-                reply = entity_domain
+            if isinstance(entity, dict):
+                logging.warning("Received unknown entity")
```

Comment:
Prev message was `"Received unknown attachment type"`, change this to `"Received unknown entity type"`

Reply:
Fix.